bench_test.go

```diff
@@ -118,3 +119,26 @@ func BenchmarkAmount_Cmp(b *testing.B) {
 	}
 	cmpResult = z
 }
+
+func BenchmarkAmount_RoundTo(b *testing.B) {
+	x, _ := currency.NewAmount("34.9876", "USD")
+
+	roundingModes := []currency.RoundingMode{
+		currency.RoundHalfUp,
+		currency.RoundHalfDown,
+		currency.RoundUp,
+		currency.RoundDown,
+	}
+
+	for i := range roundingModes {
```

Comment:
Same comment here as in the test, we can do "for _, mode := range roundingModes {"